packages/site/src/main.css

```diff
@@ -40,6 +40,10 @@ a:visited {
   color: var(--color-interactive--hover);
 }
 
+custom-elements {
+  min-height: 100dvh;
```

Comment:
In my [most recent PR](https://github.com/GetJobber/atlantis/pull/2264) I take care of this in the [BaseView](https://github.com/GetJobber/atlantis/blob/c60ffb815fbe59caefc7f7323de6352491166b82/packages/site/src/layout/BaseView.tsx#L5) I added, but I don't remind removing this when I merge mine in though if it is important to go in in this PR

Reply:
not important for this PR in particular!

Reply:
Since I am not requesting any other changes you can leave this in and I will handle the conflicts on my PR once it is in